source/manual/business-readiness-publish-changes.html.md

```diff
@@ -0,0 +1,45 @@
+---
+owner_slack: "#govuk-platform-health"
+title: Publishing changes to the business readiness finder
+section: Business readiness finder
+layout: manual_layout
+parent: "/manual.html"
+important: true
+last_reviewed_on: 2019-01-18
+review_in: 3 months
+---
+
+*If you are trying to update the content returned by the business readiness finder, see: [Updating content for the business readiness finder](/manual/business-readiness-update-content)*
+
+The full journey for the  [business readiness finder][business-readiness-finder] is made up of the following content items:
+
+1. The start page: [`/business-uk-leaving-eu`](https://www.gov.uk/business-uk-leaving-eu)
+2. The Q&A: [`/prepare-business-uk-leaving-eu`](https://www.gov.uk/prepare-business-uk-leaving-eu)
+3. The finder: [`/find-eu-exit-guidance-business`](https://www.gov.uk/find-eu-exit-guidance-business)
+
+The content for the start page can be updated in Mainstream Publisher as you would any other start page.
+
+The content item for the Q&A doesn't actually contain any details. The titles of the questions are defined in a YAML file in [finder frontend](finder-frontend). However the body of the question, for example the options if the question is a checkbox, are read from the content item of the finder itself (content item 3 in the list). Finder frontend searches the content item for a facet "key" that matches the question, e.g. `sector_business_area` to find the question content.
+
+The content item for the finder is updated from a [YAML file](govuk-app-deployment-secrets). Any changes to this YAML file affect both the facets in the finder and the options in the Q&A.
+
+## Updating question titles
+
+1. Update the content of the YAML file in [finder frontend](finder-frontend)
+2. Merge and deploy the changes
+
+## Updating the business readiness finder
+
+The content item for the business readiness finder is published by rummager. However the actual config is read in from [govuk-app-deployment-secrets](govuk-app-deployment-secrets).
+
+1. Update [govuk-app-deployment-secrets](govuk-app-deployment-secrets)
+2. Merge and deploy the change to [govuk-app-deployment-secrets](govuk-app-deployment-secrets)
+3. Run the `publishing_api:publish_document_finder` [rake task][staging-rake-task] in rummager to publish the changes:
+
+    ![download](images/publish-business-readiness.png)
+
```

Comment:
I think you also need to reindex elasticsearch if you go about changing the schema (https://docs.publishing.service.gov.uk/manual/reindex-elasticsearch.html#header)

Reply:
@bevanloon Is there any way to test this? I have a PR open: https://github.com/alphagov/govuk-app-deployment-secrets/pull/45 and I haven't figured out how to test it in integration short of merging in the change.

Reply:
hmm - not sure - could you set it up to take a branch hereish https://github.com/alphagov/govuk-puppet/commit/7dd5e6dccbc9d53dca8b43ca2486255448bf15ff?

just a guess though at a very brief look at the code for it.